Content.Client/CharacterInterface/CharacterInterfaceSystem.cs

```diff
@@ -121,6 +121,7 @@ private void HandleOpenCharacterMenu()
         private void _setOpenValue(SS14Window menu, bool value)
         {
             _gameHud.CharacterButtonDown = value;
+            menu.SetSize = (545, 400);
```

Comment:
This should actually be set wherever the SS14Window is created. The MinSize somewhere.